app/services/deploy_service.rb

```diff
@@ -1,7 +1,12 @@
 class DeployService
-  def self.start(deploy_strategy)
+  def self.start(deploy_strategy, slack_token = nil)
+    Slack.configure do |config|
+      config.token = slack_token if slack_token
+    end
```

Comment:
I think this whole `Slack.configure` block can be removed, and the `slack_token` argument eliminated from this method. Everything else should work as-is.

Excited to see this in action!